push/gradle.properties

```diff
@@ -0,0 +1,3 @@
+POM_NAME=AeroGear Android SDK
+POM_ARTIFACT_ID=push
```

Comment:
Let's use aerogear-push like in Matthias PR https://github.com/aerogear/aerogear-android-sdk/pull/144